src/components/application_manager/src/commands/hmi/request_from_hmi.cc

```diff
@@ -43,7 +43,7 @@ RequestFromHMI::RequestFromHMI(const MessageSharedPtr& message,
     : CommandImpl(message, application_manager)
     , EventObserver(application_manager.event_dispatcher()) {
   // Replace HMI app id with Mobile connection id
-  ReplaceHMIByMobileAppId(*(message.get()));
+  ReplaceHMIWithMobileAppId(*(message.get()));
```

Comment:
`*message` is used instead of `*(message.get())`, is there a difference.

Reply:
@jacobkeeler Fixed in 88fc4e0d5a15d60bd2edfc9bd623a54bd71d0243